plaidml/core/CMakeLists.txt

```diff
@@ -1,13 +1,31 @@
+set(_HDRS
+  core.h
+  ffi.h
+)
+
+set(_SRCS
+  ffi.cc
+  settings.cc
+  version.cc
+)
+
 pml_cc_library(
   NAME core
+  TYPE STATIC
+  HDRS ${_HDRS}
+  SRCS ${_SRCS}
+  DEPS
+    pmlc::ast
+    pmlc::compiler
+    pmlc::util
+    LLVMSupport
+)
+
+pml_cc_library(
```

Comment:
I wonder if there is some way to refactor this into a 'pml_cc_with_obj' or something to avoid the duplication, but probably not a huge deal given there are only 4 places this pattern seems to be used.

Reply:
Good idea, I'll give it some thought